src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationService.scala

```diff
@@ -42,7 +42,8 @@ trait MethodConfigurationService extends HttpService with PerRequestCreator with
               val extReq = Post(FireCloudConfig.Rawls.copyFromMethodRepoConfigUrl, copyMethodConfig)
               externalHttpPerRequest(requestContext, extReq)
             }
-          }
+          } ~
+          complete(StatusCodes.MethodNotAllowed, ErrorReport(StatusCodes.MethodNotAllowed, "HTTP method not allowed"))
```

Comment:
Doesn't spray respond appropriately with a 405 if the path is handled, but method is not? Or are we capturing it here to filter it through ErrorReport? If it's the latter, I think it would be a lot less boilerplate code if we instead customized the default RejectionHandler.